common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -282,3 +282,7 @@
     @Html(common.Assets.css.amp)
 
 </style>
+
+@if(play.Play.isDev() || !InlineCriticalCss.isSwitchedOn) {
```

Comment:
Would you ever want AMP to *not* inline the CSS? Say we decided suddenly that the website needed inline css switched off, we'd break AMP too so I think you probably just need the `isDev`?